public/index.html

```diff
@@ -77,6 +72,65 @@ <h1>Questions?</h1>
       <img src="./assets/images/[DSC-L] Colored Horizontal (2) 1.png" alt="DSC Logo" class="DSC-logo">
       <p>&copy; 2020 Developer Student Clubs Loyola. All Rights Reserved.</p>
     </footer>
+	
+<!----MODAL----->
+	<div class="modal-cont">
+		<div class="modal-hold">
+			<div class="modal-coverimg">
+				<img src="https://placehold.co/320x320/000000/000000/png">
+				<div class="modal-close">
+					<!--modal close-->
+					<a href="#">
+						<div id="close1"></div>
+						<div id="close2"></div>
+					</a>
+				</div>
+				<div class="modal-imgshare">
+					<!--blue share click-->
+					<a href="#" target="_blank">
+						<div id="base"></div>
+						<div class="share-dots">
+							<div id="dot1"></div>
+							<div id="dot2"></div>
+							<div id="dot3"></div>
+						</div>
+						<div class="share-lines">
+							<div id="line1"></div>
+							<div id="line2"></div>
+						</div>
+					</a>
+				</div>
+			</div>
+			<div class="modal-notimgcont">
+				<div class="modal-text">
+					<h4>Episode 1: Seizing the Opportunity to Lead in DSC</h4>
+					<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>
+				</div>
+				<div class="modal-player">
+					<div class="player-text">
+						<span>S01E01</span>
+						<p>Seizing the Opportunity to Lead</p>
+						<span>Bruce Alturas</span>
+					</div>
+					<div class="player-img">
+						<div id="player-bg"></div>
+						<div class="player-btn">
+							<div id="player-btncircle"></div>
+							<div id="player-btnsymb"></div>
+						</div>
+					</div>
+				</div>
+				<div class="modal-speaker">
+					<h5>Speaker</h5>
+					<img id="speaker-img" src="https://placehold.co/200x200/000000/000000/png">
+					<p>Bruce Alturas</p>
+					<div class="speaker-shareicons">
+						<a href="https://www.facebook.com/dscloyola" target="_blank" rel="noopener noreferrer"><img src="./assets/images/Facebook.png" alt="Facebook Icon" class="social-media-icons"></a><a href="https://twitter.com/dscloyola" target="_blank" rel="noopener noreferrer"><img src="./assets/images/Twitter.png" alt="Twitter Icon" class="social-media-icons"></a><a href="https://www.instagram.com/dsc.loyola/" target="_blank" rel="noopener noreferrer"><img src="./assets/images/Instagram.png" alt="Instagram Icon" class="social-media-icons"></a><a href="https://www.linkedin.com/company/dsc-ateneo/" target="_blank" rel="noopener noreferrer"><img src="./assets/images/Linkedin.png" alt="Linkedin Icon" class="social-media-icons"></a>
```

Comment:
Avoid super long one-liner codes.
Change it to this format to make it look cleaner

```suggestion
	<a href="https://www.facebook.com/dscloyola" target="_blank" rel="noopener noreferrer">
	   <img src="./assets/images/Facebook.png" alt="Facebook Icon" class="social-media-icons">\
	</a>
	<a href="https://twitter.com/dscloyola" target="_blank" rel="noopener noreferrer">
	    <img src="./assets/images/Twitter.png" alt="Twitter Icon" class="social-media-icons">
	</a>
	<a href="https://www.instagram.com/dsc.loyola/" target="_blank" rel="noopener noreferrer">
	    <img src="./assets/images/Instagram.png" alt="Instagram Icon" class="social-media-icons">
	</a>
	<a href="https://www.linkedin.com/company/dsc-ateneo/" target="_blank" rel="noopener noreferrer"> 
            <img src="./assets/images/Linkedin.png" alt="Linkedin Icon" class="social-media-icons">
        </a>
```